lib/utils/isAutoprefixable.js

```diff
@@ -2,7 +2,9 @@
 'use strict';
 
 const autoprefixer = require('autoprefixer');
+// @ts-ignore
```

Comment:
maybe provide some explanation why we ignore this, better in the same way `TODO TYPES <text>`

By the way for `autoprefixer/lib/browsers` and `autoprefixer/lib/prefixes` we can provide types, same as I did for `postcss/*`

Reply:
Sorry, I didn't explain good enough. The list of files that I provided, I was hopping that you remove all ignores and fix problems :) I didn't know how to fix, so I ignored things.

Reply:
I will check problematic places later

Reply:
They might not be problematic. It's just the first time I'm adding types, and I don't know a lot of things :)